tests/transformer/test_operators.py

```diff
@@ -0,0 +1,356 @@
+from RestrictedPython._compat import IS_PY35_OR_GREATER
+from tests import c_eval
+
+import pytest
+
+
+# Arithmetic Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Add(c_eval):
```

Comment:
I'd suggest to use `e_eval` here it compiles the code, assures that there are no errors and executes the code. This leads to much less code repeating the same checks over and over again in this module.